sdk/monitor/monitor-opentelemetry/src/logs/handler.ts

```diff
@@ -55,14 +56,29 @@ export class LogHandler {
    * Start auto collection of telemetry
    */
   private _initializeInstrumentations() {
-    if (this._config.instrumentationOptions.bunyan?.enabled) {
+    const bunyanLogLevelEnv =
```

Comment:
Is BUNYAN_LOG_LEVEL an env variable supported in Bunyan package?

Reply:
Removing logging library specific env vars per: https://github.com/Azure/azure-sdk-for-js/pull/29619#discussion_r1594326912